tests/middleware/test_message_state.py

```diff
@@ -0,0 +1,34 @@
+import remoulade
+from remoulade.state.message_state import MessageState
+
+
+class TestMessageState:
+    """Class to test the middleware
+    MessageState
+    """
+
+    @staticmethod
+    @remoulade.actor
```

Comment:
Nice idea to put that here !

Reply:
Maybe there is a way, to have a pytest fixture which yield an actor declared to a broker. That would be super nice, we can keep this idea for another MR though.